src/common/hooks/permissions/useHasPermission.ts

```diff
@@ -36,6 +36,10 @@ type PurchaseOrderPermissions =
   | 'create_purchase_order'
   | 'view_purchase_order'
   | 'edit_purchase_order';
+type TaskSchedulePermissions =
+  | 'create_task_schedule'
+  | 'view_task_schedule'
+  | 'edit_task_schedule';
 
```

Comment:
@turbo124 Please let me know if those specified permissions are correct for schedules.

Reply:
@Civolilah this is admin/owner only.

Reply:
@turbo124 Thankss, in that case I just changed conditions for enabling schedules queries.